bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/BandsList.scala

```diff
@@ -10,9 +8,9 @@ import Scalaz._
  * It is used, e.g. to extract a magnitude from a target
  */
 sealed trait BandsList {
-  val bands: NonEmptyList[MagnitudeBand]
-  def extract(t: SiderealTarget) = bands.map(t.magnitudeIn).list.find(_.isDefined).flatten
-  def bandSupported(b: MagnitudeBand) = bands.list.contains(b)
+  val bands: List[MagnitudeBand]
```

Comment:
Agreed that `NEL`s are a good thing conceptually, but the fact that they're not serializable seems a bit of a design flaw(?). The `Obs` objects in the `MiniModel` contain `AgsAnalysis` objects which in turn contain `BandsList`s. It seems this was added in order to allow selecting observations with certain guide star properties; @sraaphorst probably knows the details.